pkg/cmd/clientBuild_test.go

```diff
@@ -0,0 +1,251 @@
+package cmd
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func TestNewClientBuildsCmd(t *testing.T) {
+	tests := []struct {
+		name string
+		want *ClientBuildsCmd
+	}{
+		{
+			name: "test new client builds command",
+			want: &ClientBuildsCmd{},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			if got := NewClientBuildsCmd(); !reflect.DeepEqual(got, tc.want) {
+				t.Errorf("NewClientBuildsCmd() = %v, want %v", got, tc.want)
+			}
+		})
+	}
+}
+
+func TestClientBuildsCmd_GetClientBuildsCmd(t *testing.T) {
+	tests := []struct {
+		name string
+		cbc  *ClientBuildsCmd
+		want *cobra.Command
+	}{
+		{
+			name: "test get client builds command",
+			cbc:  &ClientBuildsCmd{},
+			want: &cobra.Command{
+				Use:   "clientbuilds",
+				Short: "get clientbuilds for a mobile client",
+				RunE: func(cmd *cobra.Command, args []string) error {
+					return nil
+				},
+			},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			got := tc.cbc.GetClientBuildsCmd()
```

Comment:
What if here instead of tc.cbc.GetClientBuildsCmd(), you could call something like
tc.cbcMethodCall() and instead of 

````
[]struct {
		name string
		cbc  *ClientBuildsCmd
		want *cobra.Command
}
````

you'd have


````
[]struct {
		name string
		cbcMethodCall  func() (interface{})
		want *cobra.Command
}
````
I am not entirely sure about the function return type, of if you need a funciton pointer in there, but then you could have the struct look like

````
{
name: "test get client builds command",
cbcMethodCall:  func () interface {} {
  return ClientBuildsCmd.GetClientBuildsCmd()
}
````

or something like that. That would make the data-driven testing possible.